### ISSUE TITLE
[BUG] - Tokens with deleted actors throws errors

### ISSUE THREAD

**Issue by ravingdragoon** [fantasycalendar/FoundryVTT-ItemPiles#14]
**Describe the bug**
A console error while waiting for the canvas to load.

**To Reproduce**
When running this module I get the following error twice while waiting for the canvas to load.

**Expected behavior**
No error

**Error**
```
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'getFlag')
[Detected 1 package: item-piles]
    at Module.getItemPileData (lib.js:113)
    at Function.isValidItemPile (api.js:1503)
    at Function._initializeItemPile (api.js:1606)
    at _canvasReady (module.js:94)
getItemPileData	@	lib.js:113
isValidItemPile	@	api.js:1503
_initializeItemPile	@	api.js:1606
_canvasReady	@	module.js:94
await in _canvasReady (async)		
_call	@	foundry.js:294
call	@	foundry.js:279
draw	@	foundry.js:19879
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 1.0.4

**Comment by Haxxer** [comment]
What systems and other modules are you using?

**Comment by ravingdragoon** [comment]
Foundry v9, DnD 5e 1.5.6

I get the error once when only running the modules below:
![image](https://user-images.githubusercontent.com/72875320/148428263-ed2547ae-579b-4101-b30a-901419e5c1f9.png)

**Comment by Haxxer** [comment]
Could it be that you have tokens on your canvas that no longer has an actor?

**Comment by ravingdragoon** [comment]
I don't have any item pile tokens on the canvas, if that is what you mean.

**Comment by Haxxer** [comment]
No, just any token. Try double clicking on each token in the scene, and see if you get any errors.

**Comment by ravingdragoon** [comment]
Well, it looks like I don't get the error if I am on my landing page (no tokens at all)

**Comment by Haxxer** [comment]
This is fixed in 1.0.5
